vignettes/SEIRD_Model_Compartments.Rmd

```diff
@@ -0,0 +1,249 @@
+* $\eta_{E->I_i}$ - the fraction of exposed individuals that, after the incubation period, became infectious of symptom group $i$. $\sum_i\eta_{E->I_i}=1$, $i\in\{a, m, s\}$.
+* $\gamma_{I_i}$ - the rate at which individuals of symptom group $i$ recover from the disease ($i\in\{a, m, s\}$).
+* $\mu_{I_i}$ - the rate at which individuals of symptom group $i$ die of the disease ($i\in\{a, m, s\}$).
+
+In addition to these six main ODEs above, the model also keeps track the number of 
+disease-related deaths ($D$) and the total number of cases ($C$) over time: $$\frac{\text{d}D}{\text{d}t} = \mu_{I_a} I_{a} +  \mu_{I_m}I_{m} +  \mu_{I_s} I_{s}$$, 
+$$\frac{\text{d}C}{\text{d}t} = S (\beta_{I_a}I_a + \beta_{I_m}I_m + \beta_{I_s}I_s)$$.
+
+The system is closed by specifying the initial conditions
+$S(0) = S_0,\ E(0) = E_0,\ I_a(0) = I_{a,0},\ I_m(0) = I_{m,0},\ I_s(0) = I_{s,0},\ R(0) = R_0,\ C(0) = C_0,\ D(0) = D_0.$
+
+In the package implementation, the model does not include natural death or birth, and the total population size is normalised to 1 at each time point, i.e. 
+$S(t) + E(t) + I_a(t) + I_m(t) + I_s(t) + R(t) + D(t) \equiv 1$ for any given $t$, so we require $S_0 + E_0 + I_{a,0} + I_{m,0} + I_{s,0} + R_0 = 1$, where we assume that there is initially zero disease-related deaths.
+
+
+
+## Basic reproduction number
+The basic reproduction number, $R_0$, is widely used metric in epidemiological modelling to tell whether or not a disease outbreak will occur. The value of $R_0$ is the number of individuals that an infectious person will infect in a population where everyone is susceptible. If $R_0>1$, the infectious individual will infect more than one other individuals and an epidemic will occur. If $R_0<1$, no epidemic occurs. In this section, we compute the value of $R_0$ for SEIaImIsRD model. For more mathematical detail on the method used, we refer the interested reader to [van den Driessche (2017)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6002118/pdf/main.pdf).
+
+We calculate the basic reproduction number ($R_0$) using the next generation matrix approach. First, we select those populations in the model that contain exposed or infectious individuals, so $E$, $I_a$, $I_m$ and $I_s$. We define $x = [E, I_a, I_m, I_s]$ and rewrite the ODEs of the variables in $x$ to be of the form $$\frac{\text{d}x_i}{\text{d}t} = \mathscr{F_i}(x) - \mathscr{V_i}(x)$$, where $\mathscr{F_i}(x)$ is the rate of appearance of new infections in compartment $i$, and $\mathscr{V_i}(x)$ is the rate of other transitions between compartment $i$ and other infected compartments. For example, if we take $x_i = E$, then $\mathscr{F_i}(x) = S (\beta_{I_a}I_a + \beta_{I_m}I_m + \beta_{I_s}I_s)$ and $\mathscr{V_i}(x) = \kappa E$. Similarly, for $x_i = I_a$ $\mathscr{F_i}(x) = \eta_{E->I_a} \kappa E$ and $\mathscr{V_i}(x) = (\gamma_{I_a} + \mu_{I_a})I_{a}$.
```

Comment:
For the dxi_dt equation, can we put blank between it and the text otherwise it's hard to see where it appears. Also, can we put the comma within the dollar signs